frontend/src/features/contacts/contact/detail/Person.tsx

```diff
@@ -37,8 +37,8 @@ interface AddressField {
 const PersonView: React.FunctionComponent<PersonViewProps> = ({ person }) => {
   const phoneTypes: Dictionary<string> = {};
   phoneTypes[ContactMethodTypes.WorkMobil] = 'Mobile';
-  phoneTypes[ContactMethodTypes.WorkPhone] = 'Home';
-  phoneTypes[ContactMethodTypes.PersPhone] = 'Work';
+  phoneTypes[ContactMethodTypes.WorkPhone] = 'Work';
+  phoneTypes[ContactMethodTypes.PersPhone] = 'Home';
```

Comment:
It would be a good idea to confirm with Kartik the mapping for the phones.
The db mapping was kind of ambiguous
https://confluence.th.gov.bc.ca/display/PIMS/READ+-+DETAIL+VIEW+of+Individual+In+Contact+Manager

Reply:
@FuriousLlama - will do, thanks!

Reply:
Good call on confirming, apparently WORKMOBIL = Work and PERSMOBIL = Mobile. Was definitely ambiguous on the Confluence page

Reply:
I am going to contact him again about this as that would mean personal mobile text precedence over work mobile which does not make sense to me..

Reply:
Dropping that convo entirely as it is changing the requirements and not a bug.